compose/gateway.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.6'
+
+services:
+  gateway:
+    image: quay.io/datawire/ambassador:0.39.0
+    restart: always
+    depends_on:
+      - proxy
+      - peatio
+      - barong
+      - ranger
```

Comment:
Why ambassador depends_on: peatio, barong ?
I'm not sure just asking

Reply:
Because ambassador is only gateway, that does nothing if barong and peatio is not deployed.

Reply:
@shal Peatio and Barong are depent on Ambassador because
Ambassador can work correctly without peatio & barong but peatio & barong can't work without Ambassador as I know